lib/aws_codegen.ex

```diff
@@ -90,7 +90,7 @@ defmodule AWS.CodeGen do
         end
       )
 
-    Enum.each(tasks, fn task -> Task.await(task) end)
+    Enum.each(tasks, fn task -> Task.await(task, 60000) end)
```

Comment:
Had to increase this value as sometimes generation took longer than the default of 5000 ms